flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/functions/casting/MapAndMultisetToStringCastRule.java

```diff
@@ -107,6 +120,28 @@ protected String generateCodeBlockInternal(
         final String keyArrayTerm = newName("keys");
         final String valueArrayTerm = newName("values");
 
+        return generateMapToString(
```

Comment:
No need to have this method splitting anymore

Reply:
yes, since we've decided to go with only class both for map and multiset this extraction does not make sense.
done